packages/fast-element/docs/building-components.md

```diff
@@ -0,0 +1,75 @@
+# Building Components
+
```

Comment:
This is redundant with README.md - needed in this section?

Reply:
I'm writing this document so that it can also form part of the published docs. Since the readme won't be present in that context, I wanted to make sure there was a short intro in this document as well.